cotstants.go

```diff
@@ -32,6 +33,9 @@ const (
 	helpDefaultValue     = false
 	defaultIndent        = 2
 	defaultColumnPadding = 4
+	labelCommands        = `Commands:`
```

Comment:
These constants add very little value. The general rule I use for constants is similar to what [goconst](https://github.com/jgautheron/goconst) uses, which is something like "if a constant is > N characters and is repeated > M times", consider making normalising it, where N is ~ 5 and M is ~3.